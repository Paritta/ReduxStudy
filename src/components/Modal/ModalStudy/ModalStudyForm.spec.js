import React from "react";
import { ModalStudyForm } from "./ModalStudyForm";
import "jest-styled-components";
import ReactTestRenderer from "react-test-renderer";
import configureStore from "redux-mock-store";
import { Provider } from "react-redux";

const middlewares = [];
const mockStore = configureStore(middlewares);

const initialState = {};
const store = mockStore(initialState);

it("ModalStudyForm render correctly", () => {
    const tree = ReactTestRenderer.create(
    );
    expect(tree.toJSON()).toMatchSnapshot();
});
