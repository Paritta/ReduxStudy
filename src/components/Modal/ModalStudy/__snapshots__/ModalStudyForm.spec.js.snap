// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalStudyForm render correctly 1`] = `null`;
